setup.py

```diff
@@ -201,6 +201,9 @@ def configure(self, ext: CMakeExtension) -> None:
         else:
             # Default build tool to whatever cmake picks.
             build_tool = []
+        # Make sure we use the nvcc from CUDA_HOME
+        if _is_cuda():
+            cmake_args += [f'-DCMAKE_CUDA_COMPILER={CUDA_HOME}/bin/nvcc']
```

Comment:
did you use some other nvcc instead?

Reply:
Yes, the default CUDA version is still 12.0 on our devgpus. We need >=12.3 to use flash attention 3.